packages/maskbook/src/web3/types.ts

```diff
@@ -11,7 +11,7 @@ export enum CurrencyType {
 }
 
 export interface MetaMaskInpageProvider extends ReturnType<typeof createMetaMaskProvider> {
-    _metamask?: { isUnlocked: () => Promise<boolean> }
+    _metamask: { isUnlocked: () => Promise<boolean> }
```

Comment:
@zhouhanseng It's OK?

Reply:
Sorry, I'm late. Yes now the `?` should be removed.
Before @metamask/inpage-provider@7.0.0 defines the type at `index.d.ts`, which does not include  `_metamask`.
Now @metamask/inpage-provider@8.0.3 defines the type at `dist/index.d.ts` which add `_metamask`,

Reply:
Sorry I'm late. Yes! `?` should be removed now.

Previously inpage-provider@7.0.0 defines types at `index.d.ts` which does not include `_metamask `. Now inpage-provider@8.0.3 defines types at `dist/index.d.ts` which include it.